RecoEgamma/EgammaElectronProducers/plugins/GsfElectronProducer.cc

```diff
@@ -249,6 +288,59 @@ void GsfElectronProducer::fillDescriptions(edm::ConfigurationDescriptions& descr
           "RecoEgamma/ElectronIdentification/data/TMVA_BDTSoftElectrons_7Feb2014.weights.xml",
       });
 
+  {
+    edm::ParameterSetDescription psd1;
+    psd1.add<bool>("enabled", false);
+    psd1.add<std::string>("inputTensorName", "FirstLayer_input");
+    psd1.add<std::string>("outputTensorName", "sequential/FinalLayer/Softmax");
+    psd1.add<std::string>("logLevel", "2");  //0 debug, 1 info, 2 warning
```

Comment:
1. it would be preferrable to rely on standard CMSSW EDM logging levels (LogWarning, LogInfo, LogDebug), so that things are consistent across different modules
2. if a custom logging solution is really required for EGamma (this should be discussed in e.g. core software IMO), please use at least ints for the level, not strings

Reply:
The option is there only for the Tensorflow module internal logging.  We switched to integer (adding a to_string for the tensorflow::setLogging method constructor.  https://gitlab.cern.ch/mrieger/CMSSW-DNN#logging)